doc/api/packages.md

```diff
@@ -4,11 +4,13 @@
 
 ## Introduction
 
-A package is a folder described by a `package.json` file.
+A package is a folder tree described by a `package.json` file. The package
+consists of the folder containing the `package.json` file and all subfolders
+until the next folder containing another `package.json` file, or a folder
+named `node_modules`.
 
-A folder containing a `package.json` file, and all subfolders below that folder
-until the next folder containing another `package.json` file, are a _package
-scope_.
+This section provides guidance for package authors writing `package.json` files
```

Comment:
When I read “section” I think the immediate section, like “Introduction” in this case; but what you really mean is the whole page, e.g. `packages.md`. Is there a better word to use?

Reply:
I guess we could just use page?

```suggestion
This page provides guidance for package authors writing `package.json` files
```